.docker/php/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:8.0-fpm
+
+# Arguments defined in docker-compose.yml
+ARG user
+ARG uid
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libicu-dev \
+    libpq-dev \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    libzip-dev \
+    zip \
+    unzip \
+    nodejs \
+    npm
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install PHP extensions
+RUN docker-php-ext-install intl zip pgsql pdo_pgsql mbstring exif pcntl bcmath gd
+
+# Install xdebug and redis extenstion
+RUN pecl install xdebug
+RUN pecl install redis
+
+# Enable PHP extensions
+RUN docker-php-ext-enable xdebug redis
+
+# Get latest Composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
```

Comment:
😱 cool that you can pull/copy from other images. 🤯🥳

Reply:
Yep